crates/locutus-node/src/http_gateway.rs

```diff
@@ -52,45 +56,54 @@ impl HttpGateway {
             response_channels: HashMap::new(),
         };
 
-        let get_home = warp::path::end().and_then(home);
-        let base_web_contract = warp::path::path("contract").and(warp::path::path("web"));
+        // let get_home = warp::path::end().and_then(home);
```

Comment:
remove all the comments if is no necessary anymore

Reply:
deleted